free-3.md

```diff
@@ -57,12 +87,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think there will be a lot of language decisions -- the input I need only requires 4 basic pieces of information from the user (key signature, clef, tempo in BPM, and time signature). None of the semantics of the language will look very code-y at all! 
 
```

Comment:
I think the most important language decisions will be what keys correspond to what notes.  Also how will differentiate between notes like F sharp or G flat?  Will they be different keys?  I was also wondering how intricate you imagine the systems aspect to be.  Will the notes appear in live time, as you type them in?  Will it play the notes as you type them?  Or will you be able to here the metronome?